libs/components/src/accordion/vl-accordion.component.ts

```diff
@@ -23,14 +23,15 @@ declare const vl: any;
  * @property {PADDINGS} data-vl-content-padding - De grootte van de padding van de content. Deze padding wordt toegepast op zowel desktop als mobile.
  * @property {boolean} data-vl-bold - Beeldt de toggle-text van de accordion af in bold.
  * @property {boolean} data-vl-disabled - Schakelt het openen en het sluiten van de accordion uit.
+ * @property {boolean} data-vl-default-open - Indien gezet is de accordion standaard geopend.
  * @property {string} data-vl-icon - Icoon dat getoond wordt voor de tekst van de toggle.
  * @event vl-on-toggle - Afgevuurd bij het openen en sluiten van de accordion. Het event bevat of de accordion geopend of gesloten is.
  */
 
 @webComponent('vl-accordion')
 export class VlAccordionComponent extends BaseElementOfType(HTMLElement) {
     static get _observedAttributes() {
-        return ['toggle-text', 'open-toggle-text', 'close-toggle-text', 'content-padding'];
+        return ['toggle-text', 'open-toggle-text', 'close-toggle-text', 'content-padding', 'default-open'];
```

Comment:
In principe moet je deze ook niet in de observedAttributes zetten aangezien er enkel naar gekeken wordt in de connectedCallback.

Reply:
Done!